### PR TITLE
145 fine tune styles agreement

### PR DESCRIPTION
**Pull request by yishain11**

### CODE REVIEW

File: frontend/src/pages/AgreementPage.tsx [hunk 1]
```diff
@@ -54,37 +53,40 @@ const AgreementPage: FC = () => {
...
           direction="row"
           alignItems="center"
           justifyContent="space-between"
-          sx={{ margin: '2rem' }}
+          marginTop="2rem"
+          maxWidth="md"
         >
           <Stack direction="row" alignItems="center" justifyContent="space-between">
-            <Typography variant="h1">
+            <Typography variant="h1" marginRight="1rem">
```
Comment (avirankatz) [inline frontend/src/pages/AgreementPage.tsx:65]:
```suggestion
            <Typography variant="h1" marginRight={2}>
```

File: frontend/src/pages/AgreementPage.tsx [hunk 2]
```diff
@@ -54,37 +53,40 @@ const AgreementPage: FC = () => {
...
         </Breadcrumbs>
       </Stack>
-      <Stack direction="column">
+      <Stack direction="column" spacing={4}>
         <Stack
           direction="row"
           alignItems="center"
           justifyContent="space-between"
-          sx={{ margin: '2rem' }}
+          marginTop="2rem"
```
Comment (avirankatz) [inline frontend/src/pages/AgreementPage.tsx:61]:
```suggestion
          marginTop={4}
```

**Review by avirankatz — CHANGES_REQUESTED** [review]
Nice PR!
- [ ] switch margins to numbers (see suggestions)

File: frontend/src/pages/AgreementPage.tsx (hunk 1 above)
Comment (yishain11) [reply frontend/src/pages/AgreementPage.tsx:65]: done

**Pull request merged**